src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Windows.cs

```diff
@@ -279,18 +283,40 @@ protected virtual void UpdateEmptyView()
 
 		protected virtual void UpdateItemsLayout()
 		{
-			ListViewBase.IsSynchronizedWithCurrentItem = false;
+			if (ListViewBase is FormsGridView gridView)
+			{
+				if (Layout is LinearItemsLayout linearItemsLayout)
+				{
+					gridView.Orientation = linearItemsLayout.Orientation == ItemsLayoutOrientation.Horizontal
+						? Orientation.Horizontal
+						: Orientation.Vertical;
```

Comment:
Duplicated code.

Reply:
Updated to use a created `ToPlatform` extension method.